tests/keyinput/test_basekeyparser.py

```diff
@@ -63,44 +63,26 @@ class TestSplitCount:
 
     """Test the _split_count method.
 
-    Attributes:
-        kp: The BaseKeyParser we're testing.
+    Class Attributes:
+        TESTS: list of parameters for the tests, as tuples of
+        (input_key, supports_count, expected)
     """
 
-    @pytest.fixture(autouse=True)
-    def setup(self):
-        self.kp = basekeyparser.BaseKeyParser(0, supports_count=True)
-
-    def test_onlycount(self):
+    TESTS = [
```

Comment:
Added a comment at the top, I'm not sure we would gain much by using a `namedtuple` here, it would just add to the line noise in my opinion.

See the new version and let me know what you guys think.